public/imgs/home_icon.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
i personally go for more terse names so here `home_icon.svg` could be truncated to `home.svg` but that is just me